AHBC_MIDTERM_2019_JULY_TEAMROCKET/ICard.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AHBC_MIDTERM_2019_JULY_TEAMROCKET
+{
+    interface ICard
+    {
+        void PayWithCard(double total);
+
```

Comment:
For the interface of the ICard you can add a property too store credit card number to so
string CreditCardNumber;